keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -151,49 +164,227 @@ Scenario: [...]
+    project will have direct access to the key, mitigating one of the main risks of this proposal
+- We want to avoid managing the infrastructure ourselves, including buckets, mirrors/CDNs...
+- We want to provide 'as a service' access to the packages infrastructure to Release Managers and eventually other
+  Kubernetes maintainers for their subprojects
+
+#### How Open Build Service works?
+
+From the [OBS website](https://openbuildservice.org/):
+
+> The Open Build Service (OBS) is a generic system to build and distribute binary packages from sources in an automatic, consistent and reproducible way. You can release packages as well as updates, add-ons, appliances and entire distributions for a wide range of operating systems and hardware architectures.
+
+OBS works in a way that we push sources and package spec files. Upon pushing packages/changes, OBS automatically
+triggers builds for all chosen operating systems and architectures. Under the hood, OBS uses the same set of tools that
+we use for building packages: `dpkg-buildpackage` and `rpmbuild`.
+
+OBS implements a simple source-control management (SCM) system. It provides a complete history for all packages
+allowing users to see what spec files and sources we used to build the concrete package. The history is accessible
+via the OBS web interface.
+
+Interaction with the OBS platform is done mainly via the [`osc` command-line tool][osc]. Alternatively, it's possible
+to interact via the web interface. Currently there are no (Go) libraries that we can use instead of the `osc` tool.
+
+[osc]: https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.osc.html
+
+#### Packages, Operating Systems, and Architectures in Scope
+
+We'll publish Debian-based (`deb`) and RPM-based (`rpm`) packages. Packages will be published for the following
+architectures:
+
+* `x86_64` (`amd64`)
+* `aarch64`
+* `armv7l` (`arm`)
+* `ppc64le`
+* `s390x`
+
+We'll **build** packages for those architectures on the following Operating Systems:
+
+- Ubuntu 20.04 (`aarch64`, `armv7l`, `ppc64le`, `s390x`, `x86_64`)
+- CentOS Stream 8 (`aarch64`, `ppc64le`, `x86_64`)
+  - CentOS Stream 8 doesn't support `armv7l` and `s390x` on OBS, hence, we use OpenSUSE Factory which does support
+    those architectures
+- OpenSUSE Factory (`armv7l`, `s390x`)
+
+Those operating systems are used as **builders**. The resulting packages are supposed to work on any Debian-based
+and RPM-based distribution, respectively. Those distributions were chosen based on recommendations from the OBS team
+to provide the best compatibility with all all Debian-based and RPM-based distributions.
+
+The following packages will be published for all operating system and architectures listed earlier. For simplicity,
+we'll refer to those as packages as the **core packages**:
+
+- cri-tools
+- kubeadm
+- kubectl
+- kubelet
+- kubernetes-cni
```

Comment:
this is no longer required since the dockershim deprecation

Reply:
I think we still want to keep it for the compatibility reasons. To make migration as seamless as possible, we don’t want to change what packages we publish, as well as, what architectures we publish for.

Reply:
fair enough, it was more a FYI 😄